test-complete-app/gradle.properties

```diff
@@ -0,0 +1,13 @@
+mlHost=localhost
+mlUsername=admin
+mlPassword=admin
+
+mlNoRestServer=true
+
+# Since the REST API server for this sample project is configured to use a certificate template for SSL, we set this
```

Comment:
These comments don't look needed here. Even if `mlSimpleSsl` is needed at some point, we don't need all the comments copy/pasted here from an ml-gradle example project.